ci/infra/testrunner/skuba/skuba.py

```diff
@@ -50,18 +50,20 @@ def cluster_deploy(self, kubernetes_version=None, cloud_provider=None,
 
 
     @step
-    def cluster_init(self, kubernetes_version=None, cloud_provider=None):
+    def cluster_init(self, kubernetes_version=None, cloud_provider=None, path=None):
```

Comment:
We should add the `path` parameter also to the testrunner command in both the `bootstrap` and `deploy` commands, which call `cluster_init`  https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/testrunner.py#L167-L181  and https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/testrunner.py#L43-L56

And pass it when invoking the cluster init here https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/skuba/skuba.py#L43-L49 and here https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/skuba/skuba.py#L71-L74.